scripts/git/google_to_main.sh

```diff
@@ -0,0 +1,17 @@
+set -x
+set -e
+set -o pipefail
+
+GREEN_COMMIT="${1:-google}"
+BRANCH="${BRANCH:-google-to-main}"
+
+git sync
```

Comment:
Suggested replacement for `git sync` custom alias usage:

```suggestion
git status --porcelain  # or git status --porcelain --ignore-submodules
git checkout google && git pull
```

* More explicit
* Avoids potentially calling `git submodule update` multiple times (slow)
* Doesn't require a custom alias

I'd also consider dropping `GREEN_COMMIT` from the script:

* In the common case `git checkout "${GREEN_COMMIT?}"` does a redundant `git checkout google`
* If a different commit is needed, you're already off the rails and into "look closely at what you're doing" territory, and the only complicated part of this script is the PR body using `git log`

Reply:
I'm down for replacing git sync. `git submodule update --init` is important though because otherwise the submodules can get screwed up (from experience).

Also note we need to actually test the output of `git status --porcelain`. I think `--ignore-submodules` is a bad idea because that would also ignore local edits to the submodules. Basically before we start doing all this scripting I think it's good to be very sure the repo is in a stable state.

I disagree that a different green commit is off the rails though. I've used it already and it works nicely. Checking for a green commit on the source branch should be part of the normal workflow anyway.

Reply:
Updated google_to_main.sh. If that LG, I'll apply the same to the rest of the scripts.